test/test-errortracker.js

```diff
@@ -46,28 +49,15 @@ describe('Test how server responds to requests', function() {
     'debug': 1,
   };
   let randomVal = 1;
-  before(function() {
-    sinon.stub(Math, 'random').callsFake(function() {
+  beforeEach(function() {
+    stub = sandbox.stub(Math, 'random').callsFake(function() {
       return randomVal;
     });
+    sandbox.stub(log, 'write').yields(false);
```

Comment:
Nit: `nodebacks` expect either an `Error` instance or `null` as the first param.